crates/re_time_panel/src/time_selection_ui.rs

```diff
@@ -302,7 +294,7 @@ fn paint_range_text(
 ) {
     use egui::{Pos2, Stroke};
 
-    if selected_range.min <= TimeInt::MIN_TIME_PANEL {
+    if selected_range.min <= TimeInt::MIN {
         return; // huge time selection, don't show a confusing times
     }
```

Comment:
I don't think we need this anymore, since we no longer paint timeless data at `MIN_TIME_PANEL`. All painted data are at interesting positions that the user picked

Reply:
ah, nice. Was overly cautious then :)